script.js

```diff
@@ -1 +1,98 @@
-console.log("Assignment 2");
\ No newline at end of file
+
+var bookList;
+var bookListCurrent;
+
+async function fetchBooks(){
+    let books;
+    await fetch('book_list.json').then(response => {
```

Comment:
Looks like a mix of async/await approach and Promise approach.
Can try to stick to only one approach.